docs/content/en/docs/getting-started/deployment/kubernetes.md

```diff
@@ -43,6 +43,31 @@ If Tetragon does not to start due to BTF issues, please refer to the
 for details and solutions.
 {{< /note >}}
 
+## Validation
+
+Check if all the tetragon containers are running.
+
+{{< note >}}
+In the below examples, don't forget to change the Pod name corresponding to the Pod name inside your cluster.
+
+This example contains Tetragon Pod container named `tetragon-qj4w4`.
+{{< /note >}}
+
+```shell
+➜  ~ kubectl get pods --all-namespaces
+NAMESPACE            NAME                                         READY   STATUS    RESTARTS   AGE
+kube-system          coredns-5d78c9869d-kgwbw                     1/1     Running   0          12m
+kube-system          coredns-5d78c9869d-v728b                     1/1     Running   0          12m
+kube-system          etcd-test-control-plane                      1/1     Running   0          12m
+kube-system          kindnet-mwgfk                                1/1     Running   0          12m
+kube-system          kube-apiserver-test-control-plane            1/1     Running   0          12m
+kube-system          kube-controller-manager-test-control-plane   1/1     Running   0          12m
+kube-system          kube-proxy-gm8cv                             1/1     Running   0          12m
+kube-system          kube-scheduler-test-control-plane            1/1     Running   0          12m
+kube-system          tetragon-qj4w4                               2/2     Running   0          2m31s
+local-path-storage   local-path-provisioner-6bc4bddd6b-kk6hj      1/1     Running   0          12m
+```
+
```

Comment:
```suggestion
```
The The last step of the installation is already doing that more appropriately with `kubectl rollout status -n kube-system ds/tetragon -w`.

Reply:
sure.